README.md

```diff
@@ -1 +1,97 @@
-# bitrise-step-decrypt-file
\ No newline at end of file
+# Decrypt file
+
+You can use this step to decrypt a file in your repository or stored on bitrise.io or other provider (with the File Downloader step).
+
+How to create an encrypted file:
+
+1. Create a secure passphrase of at least 22 alphanumeric characters: `pwgen -s 22 1`
+2. Encrypt file: `gpg -c my_secret.txt`
+3. Upload file (`my_secret.txt.gpg`) to bitrise.io or to any other provider.
```

Comment:
If you do not explain the hosted file version of the flow please omit this line.